nipype/utils/filemanip.py

```diff
@@ -288,6 +313,10 @@ def copyfile(originalfile, newfile, copy=False, create_new=False,
     if hashmethod is None:
         hashmethod = config.get('execution', 'hash_method').lower()
 
+    # Don't try creating symlinks on CIFS
+    if _on_cifs(newfile):
```

Comment:
For performance reasons, I would only run this check if we are about to make a symlink.